app/views/channels_ui/_form_twilio.haml

```diff
@@ -9,3 +9,6 @@
 
 .row
   = f.text_field :limit, colspan: 's4', helper: 'Number of concurrent calls'
+
+.row
+  = f.text_field :base_url, label: 'Twilio base URL'
```

Comment:
```suggestion
  = f.text_field :base_url, label: 'Custom Twilio base URL'
```

Reply:
Adding what's the current default may help users tell what format the base URL should have (something like `Leave empty to default to https://api.twilio.com/2010-04-01` - so users know they don't need to add the trailing forward-slash, but they do need to include the version of the API).